src/plugins/bodhi.c

```diff
@@ -271,6 +270,36 @@ static void bodhi_print_errors_from_json(json_object *json)
     return;
 }
 
+/**
+ * Parses only name of rpm package
+ */
+static int parse_pkginfo_name(const char *pkg_full_name, char **name)
```

Comment:
with arch part, I will put the comments there, because it is not clear, thank you.